ring/ring.proto

```diff
@@ -44,6 +44,16 @@ message InstanceDesc {
 
 	// ID of the instance. This value is the same as the key in the ingesters map in Desc.
 	string id = 9;
+
+	// Timestamp when the read_only flag was updated. This is used to find other instances that could have possibly
+	// owned a specific token in the past on the write path, due to *this* instance being read-only.
+	// This value should only increase.
+	int64 read_only_updated_timestamp = 10;
```

Comment:
[nit] Can you clarify the unit (e.g. unix timestamp with seconds precision) in the field doc, please?